smalig/grammar.yaml

```diff
@@ -92,12 +92,96 @@
   format: "ØØ|op AAAA BBBB"
   format_id: "32x"
   syntax: "move-wide/from16 vAAAA, vBBBB"
-  args_info: "A: destination register pair (16 bits), B: source register pair (16 bits)"
+  args_info: "A: destination register-pair (16 bits), B: source register-pair (16 bits)"
   short_desc: "Move the contents of one register-pair to another."
-  long_desc: "Move the value of the (vBBBB, vBBBB+1) register-pair to the (vAAAA, vAAAA+1) register-pair. Both register pairs must be in the first 64k register range (0-65535) & 16 bits."
-  note: "It is legal to move from vN to either vN-1 or vN+1, so implementations must arrange for both halves of a register pair to be read before anything is written."
-  example: "" #TODO: Add example
-  example_desc: ""
+  long_desc: "Move the value of the (vBBBB, vBBBB+1) register-pair to the (vAAAA, vAAAA+1) register-pair. Both register-pairs must be in the first 64k register range (0-65535) & 16 bits."
+  note: "It is legal to move from vN to either vN-1 or vN+1, so implementations must arrange for both halves of a register-pair to be read before anything is written."
+  example: "0623 0011 - move-wide/16 v23, v17"
+  example_desc: "Moves the value from register-pair (v17, v18) to register-pair (v23, v24)."
+
+- opcode: "07"
+  name: "move-object"
+  format: "B|A|op"
+  format_id: "12x"
+  syntax: "move-object vA, vB"
+  args_info: "A: destination register (4 bits), B: source register (4 bits)"
+  short_desc: "Move the contents of one object-bearing register to another."
+  long_desc: "Move the object reference from register vB to register vA. The source register (vB) and destination register (vA) are within the 4-bit register range (0-15)."
+  note: ""
+  example: "0781 - move-object v1, v8"
+  example_desc: "Moves the object reference in v8 to v1."
+
+- opcode: "08"
+  name: "move-object/from16"
+  format: "AA|op BBBB"
+  format_id: "22x"
+  syntax: "move-object/from16 vAA, vBBBB"
+  args_info: "A: destination register (8 bits), B: source register (16 bits)"
+  short_desc: "Move the contents of one object-bearing register to another."
+  long_desc: "Move the object reference from the source register vBBBB to the destination register vAA. The source register can address 64k registers (0-65535), and the destination register can address 256 registers (0-255)."
```

Comment:
The `long_desc` (or longer description) should be as informative and simple as possible to understand. Therefore, instead of using terms like `object reference`, use `value of (if any type, then here) register...`.